test/unit/test_events.py

```diff
@@ -424,7 +432,7 @@ def test_all_serializable(self):
 
         # if we have everything we need to test, try to serialize everything
         for event in sample_values:
-            d = event_to_serializable_dict(event, lambda _: event.get_ts_rfc3339())
+            d = event_to_serializable_dict(event)
```

Comment:
just noting here for posterity that this reduction in expressiveness of `event_to_serializable_dict` is fine because it's only called by json log lines which all use the same timestamp formatting. It was originally intended to be called by log messages going to stdout too but that's unnecessary flexibility now.